.github/workflows/unit-tests.yml

```diff
@@ -113,13 +113,19 @@ jobs:
         with:
           path: ${{ github.run_id }}
 
+      - name: Check for notebook changes
+        id: check_notebooks
+        run: check_notebooks=$(git diff --quiet main HEAD -- '*.ipynb' && echo false || echo true) >> $GITHUB_ENV
+
       - name: Run tests
         env:
           BIONEMO_DATA_SOURCE: ngc
         run: ./ci/scripts/run_pytest.sh --no-nbval --skip-slow
 
       - name: Run notebook tests
-        if: ${{ contains(github.event.pull_request.labels.*.name, 'INCLUDE_NOTEBOOKS_TESTS') }}
+        if: |
+          ${{ env.STATE_check_notebooks == 'true' ||
+              contains(github.event.pull_request.labels.*.name, 'INCLUDE_NOTEBOOKS_TESTS') }}
         env:
           BIONEMO_DATA_SOURCE: ngc
         run: pytest --nbval-lax -p no:python docs/ sub-packages/
```

Comment:
cant we use here

run: `./ci/scripts/run_pytest.sh --skip-slow instead` of `pytest --nbval-lax -p no:python docs/ sub-packages/`